build.assets/Makefile

```diff
@@ -23,18 +23,14 @@ OS ?= linux
 ARCH ?= amd64
 BUILDBOX_VERSION ?= teleport11
 
-# NOTE: Windows Drone scripts hardcode $GOLANG_VERSION rather than drawing it
```

Comment:
Comments are misleading, dronegen/windows.go seems to use `make print-go-version` and `make print-node-version` now.